docs/source/advanced_features/fusion_dataset.rst

```diff
@@ -100,3 +108,9 @@ The corresponding class of frame is :class:`~tensorbay.dataset.frame.Frame`.
 Each data inside a frame corresponds to a sensor.
 And the data of the fusion dataset is the same as the data
 (:ref:`ref <reference/dataset_structure:data>`) of the dataset.
```

Comment:
data (:ref:`ref <reference/dataset_structure:data>`) -> :ref:`reference/dataset_structure:data`

catalog also needs to be modified.